src/LinSys/LinSysMerger.h

```diff
@@ -1,7 +1,7 @@
 // *****************************************************************************
 /*!
   \file      src/LinSys/LinSysMerger.h
-  \author    J. Bakosi
+  \author    Los Alamos National Laboratory
```

Comment:
Agreed. LANL as an author block is pretty not useful. As per this discussion I have fixed with PR 147 and the future ones in the cmake and TPL repos